cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -49,6 +55,58 @@ subjects:
   name: rook-ceph-cluster
   namespace: rook-ceph
 ---
+# Extra powers for the ceph-mgr daemon, as it may run a k8s client
+# for managing Rook from within.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: rook-ceph-mgr
+  namespace: rook-ceph
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-mgr
+subjects:
+- kind: ServiceAccount
+  name: rook-ceph-mgr
+  namespace: rook-ceph
+---
+# TODO: scope this down from a ClusterRole to a Role
```

Comment:
I believe you need to keep this as a `ClusterRole` since it's using cluster-wide resources such as nodes. All you should need to do is use a `RoleBinding` to make it specific to the namespace above instead of the `ClusterRoleBinding`.

Reply:
Hmm, when I try changing the ClusterRoleBinding to a RoleBinding, it can't read the configmaps in rook-ceph-system (needed to list the devices).

Possibly we need the rook-ceph-mgr service account to have an additional rolebinding associated with a read-only role in the rook-ceph-system namespace?

Reply:
Yep, that seems to work: I've updated this PR.